extensions/ql-vscode/src/run-remote-query.ts

```diff
@@ -1,17 +1,20 @@
-import { Uri, window } from 'vscode';
+import { QuickPickItem, Uri, window } from 'vscode';
 import * as yaml from 'js-yaml';
 import * as fs from 'fs-extra';
 import { getOnDiskWorkspaceFolders, showAndLogErrorMessage, showAndLogInformationMessage } from './helpers';
 import { Credentials } from './authentication';
 import * as cli from './cli';
 import { logger } from './logging';
-
+import { getRemoteRepositoryLists } from './config';
 interface Config {
   repositories: string[];
   ref?: string;
   language?: string;
 }
 
+// User-defined repository lists.
+const REPO_LISTS = getRemoteRepositoryLists();
```

Comment:
It looks like this gets the repository lists once and then it is fixed. What happens if someone changes their repository lists? Do they need to restart?

Maybe, instead of a list, call `getRemoteRepositoryLists()` every time immediately before you use it.